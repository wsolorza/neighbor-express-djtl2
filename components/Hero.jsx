const Hero = ({ block }) => (<section aria-label="Introduction">
  <div className="usa-hero" style={{ backgroundImage: `url(${block.image})` }}>
    <div className="grid-container">
      <div className="usa-hero__callout">
        <h1 className="usa-hero__heading">
          <span className="usa-hero__heading--alt">{block.title}</span>
        </h1>
        <div>
          {block.body}
        </div>
      </div>
    </div>
  </div>
</section>
)

export default Hero;