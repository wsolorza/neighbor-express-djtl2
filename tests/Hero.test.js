import { shallow } from "enzyme";
import Hero from "../components/Hero.jsx";

let wrapper;

const block = {
  title: 'Test title',
  body: 'Test body',
  image: 'https://icatcare.org/app/uploads/2019/09/The-Kitten-Checklist-1.png'
};

beforeEach(() => {
  wrapper = shallow( <Hero block={block} />);
});

describe("Hero block", () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});