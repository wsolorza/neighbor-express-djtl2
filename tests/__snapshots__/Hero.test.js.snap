// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero block renders correctly 1`] = `
<section
  aria-label="Introduction"
>
  <div
    className="usa-hero"
    style={
      Object {
        "backgroundImage": "url(https://icatcare.org/app/uploads/2019/09/The-Kitten-Checklist-1.png)",
      }
    }
  >
    <div
      className="grid-container"
    >
      <div
        className="usa-hero__callout"
      >
        <h1
          className="usa-hero__heading"
        >
          <span
            className="usa-hero__heading--alt"
          >
            Test title
          </span>
        </h1>
        <div>
          Test body
        </div>
      </div>
    </div>
  </div>
</section>
`;
